### PR TITLE
Adding sysdig docs

### PR DESCRIPTION
**Pull request by 03cranec**

### CODE REVIEW

**Comment by 03cranec** [comment]
References this new sysdig sample being added to kubernetes: https://github.com/kubernetes/kubernetes/pull/29086

File: docs/getting-started-guides/sysdig.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
+---
+
+Kubernetes comes built-in with some basic resource utilization monitoring as described at the [Monitoring](/docs/user-guide/monitoring) user guide page. Here we describe how to utilize distributed, full-stack application monitoring and deep system troubleshooting for your Kubernetes cluster with Sysdig.
```
Comment (jaredbhatti) [inline docs/getting-started-guides/sysdig.md]: "Here we describe" should be "This page describes"

File: docs/getting-started-guides/sysdig.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+### Kubernetes and Sysdig
+Sysdig offers native, first class support for Kubernetes. That includes [sysdig](http://www.sysdig.org/), an open source system troubleshooting tool, and [Sysdig Cloud](https://sysdig.com/), a monitoring solution that has been designed from the ground up to support containers and microservices.
+
+In this post we will quickly preview the Kubernetes visibility in both open source sysdig and Sysdig Cloud, covering two primary use cases:
```
Comment (jaredbhatti) [inline docs/getting-started-guides/sysdig.md]: Slightly confusing sentence.  I'd suggest replacing "In this post we will quickly preview the Kubernetes visibility in both open source sysdig and Sysdig Cloud, covering two primary use cases:" with
"This document covers two primary use cases for using open source sysdig and Sysdig Cloud to provide visibility into your Kubernetes deployments:"

File: docs/getting-started-guides/sysdig.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+- [Troubleshooting Kubernetes with sysdig](#Troubleshooting-Kubernetes-with-sysdig) - start here if you're looking for an open source tool that offers deep visibility into a single Kubernetes instance
+- [Monitoring Kubernetes with Sysdig Cloud](#Monitoring-Kubernetes-with-Sysdig-Cloud) - start here if you're looking for a commercial monitoring solution that offers metrics, dashboards, alerting, etc for a distributed Kubernetes cluster
+
+## Troubleshooting Kubernetes with sysdig 
+
+Open source sysdig is designed to offer simple, intuitive, and incredibly rich insight into all activity on a Linux machine, including inside containers. Sysdig is scriptable in Lua and includes a command line interface and a powerful interactive UI, csysdig, that runs in your terminal. Think of sysdig as strace + tcpdump + htop + iftop + lsof. With state of the art container visibility on top.
```
Comment (jaredbhatti) [inline docs/getting-started-guides/sysdig.md]: Replace "strace + tcpdump + htop + iftop + lsof. With.." with "strace + tcpdump + htop + iftop + lsof, with.."  Should be a comma.

File: docs/getting-started-guides/sysdig.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+Install sysdig on any Kubernetes node that you want to troubleshoot by following the install instructions on [github](https://github.com/draios/sysdig/wiki/How%20to%20Install%20Sysdig%20for%20Linux) or [sysdig.org](http://www.sysdig.org/install/).
```
Comment (jaredbhatti) [inline docs/getting-started-guides/sysdig.md]: I would delete "[github](https://github.com/draios/sysdig/wiki/How%20to%20Install%20Sysdig%20for%20Linux) or " and just point to the sysdig site.

**Comment by pwittrock** [comment]
@03cranec We are pretty over-committed on review bandwidth now, so it probably makes more sense for this to be hosted in a separate repo that you can control and update without going through a k8s doc review process.  We can then link to your external page.  WDYT?

**Comment by kelseyhightower** [comment]
@03cranec After careful consideration we think this kinda doc needs to live outside of the Kubernetes GitHub org for the following reasons:

- Make it easier to maintain by the document owner without the need for PRs against the main Kubernetes docs.
- Lack of bandwidth or expertise to review solutions like these and keep them up to date

Instead of having this doc live within this repo, I'm going to suggest that you host it on the sysdig website and allow us to link to it from here. This will give sysdig control over the content and allow users to discover that it exists from here. In the coming weeks we plan to push more content like this outside of the main docs repo and simply link to the external doc.

@03cranec Again thanks for your contribution, but I'm going to close this issue based on the reasoning above.

**Comment by 03cranec** [comment]
@pwittrock @kelseyhightower thanks for the feedback. Totally see where you’re coming from. I actually tried to make this doc easy to approve and maintain by design, by pulling the content from this k8s blog post (http://blog.kubernetes.io/2015/11/monitoring-Kubernetes-with-Sysdig.html) and by linking the install instructions offsite (which should be the only dynamic content). In other words, there’s no technical content here that hasn’t already been approved by the k8s team; and there is no content that should be changing or need any maintenance as sysdig/k8s evolve.

But if that doesn’t work for you, I understand, no problem.

Do you have some other structure in mind here, for linking out, so I can resubmit? Seems like it’d be a nice user experience to have at least some brief, static summary of the integration in the k8s docs, and then link out to the main content offsite (vs the jarring  experience of clicking a menu item in one context and getting sent off to another domain). I’m thinking I could trim this doc down to just one paragraph, and then link to the full content on sysdig.org from there. Maybe that could evolve over time into a list of monitoring/troubleshooting solutions that offer native support for k8s with links to offsite getting started guides for each? What to do you think?

I’m hoping this can be a vehicle not only for exposure for sysdig, but more importantly trying to help make sure that new k8s users have the best possible experience, and also build credibility for k8s in general by helping demonstrate the robust ecosystem of tools and services, open source and commercial, that are available to you when you adopt the platform. In any case, happy to follow your lead on how best to operate on this... but I would love to get this content in the docs soon. Thanks!

**Pull request closed without merging**